docs/add-ins/get-and-set-recurrence.md

```diff
@@ -0,0 +1,191 @@
+
+> [!NOTE]
+> You can also use the [`firstDayOfWeek`][firstDayOfWeek link] property with the `weekly` recurrence type. The specified day will start the list of days displayed in the recurrence dialog.
+
+## Access recurrence
+
+How you access the recurrence pattern and what you can do with it depends on if you're the appointment organizer or an attendee.
+
+**Table 2. Applicable appointment states**
+
+|Appointment state|Is recurrence editable?|Is recurrence viewable?|
+|---|---|---|
+|Appointment organizer - compose series|Yes ([`setAsync`][setAsync link])|Yes ([`getAsync`][getAsync link])|
+|Appointment organizer - compose instance|No (`setAsync` returns an error)|Yes ([`getAsync`][getAsync link])|
+|Appointment attendee - read series|No (`setAsync` not available)|Yes ([`item.recurrence`][item.recurrence link])|
+|Appointment attendee - read instance|No (`setAsync` not available)|Yes ([`item.recurrence`][item.recurrence link])|
+|Meeting request - read series|No (`setAsync` not available)|Yes ([`item.recurrence`][item.recurrence link])|
+|Meeting request - read instance|No (`setAsync` not available)|Yes ([`item.recurrence`][item.recurrence link])|
+
+## Set recurrence as the organizer
+
+Along with the recurrence pattern, you also need to determine the start and end dates and times of your appointment series. The [`SeriesTime`][SeriesTime link] object is used to manage that information.
+
+The appointment organizer can specify the recurrence pattern for an appointment series in compose mode only. In the following example, the appointment series is set to occur from 10:30 AM to 11:00 AM PST every Tuesday and Thursday during the period November 2, 2019 to December 2, 2019.
+
+```js
+var seriesTimeObject = new Office.SeriesTime();
+seriesTimeObject.setStartDate(2019,10,2);
+seriesTimeObject.setEndDate(2019,11,2);
+seriesTimeObject.setStartTime(10,30);
+seriesTimeObject.setDuration(30);
+
+var pattern = {
+    "seriesTime": seriesTimeObject,
+    "recurrenceType": "weekly",
+    "recurrenceProperties": {"interval": 1, "days": ["tue", "thu"]},
+    "recurrenceTimeZone": {"name": "Pacific Standard Time"}};
+
+Office.context.mailbox.item.recurrence.setAsync(pattern, callback);
+
+function callback(asyncResult)
+{
+    console.log(JSON.stringify(asyncResult));
+}
+```
+
+## Get recurrence
+
+### Get recurrence as the organizer
+
+In the following example, in compose mode, the appointment organizer can get the recurrence object of an appointment series given the series or an instance of that series.
```

Comment:
For consistency, consider removing "can get" and rewrite so that "the organizer gets the recurrence object..."